File: composer/src/main/java/io/piano/android/composer/model/CookieObject.java
package io.piano.android.composer.model;

import androidx.annotation.Keep;
import androidx.annotation.Nullable;

import com.google.gson.annotations.SerializedName;

@Keep
public class CookieObject {
    @Nullable
    @SerializedName("cookie_value")
    public String value;
}

File: composer/src/main/java/io/piano/android/composer/model/CookieObject.kt
package io.piano.android.composer.model

import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
public class CookieObject(
    @JvmField
    @Json(name = "cookie_value")
    public val value: String,
)
